test/integration/acceptance/tx_acceptance_test.cpp

```diff
@@ -0,0 +1,394 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//TODO: Solonets / / Satisfy disabled test
+
+#include "backend/protobuf/transaction.hpp"
+#include "builders/protobuf/queries.hpp"
+#include "builders/protobuf/transaction.hpp"
+#include "cryptography/crypto_provider/crypto_defaults.hpp"
+#include "cryptography/ed25519_sha3_impl/internal/ed25519_impl.hpp"
+#include "datetime/time.hpp"
+#include "framework/integration_framework/integration_test_framework.hpp"
+#include "integration/pipeline/tx_pipeline_integration_test_fixture.hpp"
+#include "model/generators/query_generator.hpp"
+#include "responses.pb.h"
+#include "module/shared_model/builders/protobuf/test_transaction_builder.hpp"
+
+using namespace iroha::model::generators;
+using namespace iroha::model::converters;
+
+constexpr auto kAdmin = "admin@test";
+constexpr auto kNonUser = "nonuser@test";
+constexpr auto kAsset = "coin#test";
+const auto kAdminOldKeypair = iroha::create_keypair();
+const shared_model::crypto::Keypair kAdminKeypair(
+    shared_model::crypto::PublicKey(kAdminOldKeypair.pubkey.to_string()),
+    shared_model::crypto::PrivateKey(kAdminOldKeypair.privkey.to_string()));
+auto checkStatelessValid = [](auto &status) {
+  ASSERT_NO_THROW(
+      boost::get<shared_model::detail::PolymorphicWrapper<
+          shared_model::interface::StatelessValidTxResponse>>(status.get()));
+};
+auto checkStatelessInvalid = [](auto &status) {
+  ASSERT_NO_THROW(
+      boost::get<shared_model::detail::PolymorphicWrapper<
+          shared_model::interface::StatelessFailedTxResponse>>(status.get()));
+};
+
+auto checkProposal = [](auto &proposal) {
+  ASSERT_EQ(proposal->transactions.size(), 1);
+
+};
+auto checkStatefulInvalid = [](auto &block) {
+  ASSERT_EQ(block->transactions.size(), 0);
+};
+auto checkStatefulValid = [](auto &block) {
+  ASSERT_EQ(block->transactions.size(), 1);
+};
+/**
+ * @given non existent user
+ * @when sending  transaction to the ledger
+ * @then receive STATELESS_VALIDATION_SUCCESS status
+ *       AND STATEFUL_VALIDATION_FAILED on that tx
+ */
+TEST(AcceptanceTest, NonExistentCreatorAccountId) {
```

Comment:
These are tests for AddAssetQuantity, right? Use related fixture and filename, not really good idea place all other tests in one file

Reply:
These are tests for transactions - [Transaction acceptance tests IR-907](https://soramitsu.atlassian.net/browse/IR-907) , so their placement seems acceptable.